allure-rest-assured/src/main/java/io/qameta/allure/restassured/AllureRestAssured.java

```diff
@@ -65,6 +72,16 @@ public AllureRestAssured setResponseAttachmentName(final String responseAttachme
         return this;
     }
 
+    /**
+     * Configure filter to consider headers that are blacklisted by RestAssured LogConfig. Enabled by default.
+     * @see io.restassured.config.LogConfig#blacklistHeader
+     * @see io.restassured.config.LogConfig#blacklistHeaders
+     */
+    public AllureRestAssured followHeadersBlacklist(final boolean isEnabled) {
```

Comment:
Do we need to have the possibility to override the default behaviour and expose blacklisted headers in the report?

Reply:
They are exposed now. But from my perspective - it doesn't make any sense to blacklist them in logs but expose them in Allure report.
Should I remove the configuration option and make blacklist-follow the only behavior?

Reply:
> Should I remove the configuration option and make blacklist-follow the only behavior?

Yes, let's start with that. We can introduce the configuration option later if requested by the community

Reply:
Removed configurable option - always reads log config.